### ISSUE TITLE
`schema` function needs a `schema` property to be supplied at a minimum: minimum property number not met (1)

### ISSUE THREAD

**Issue by ShepleySound** [daveshanley/vacuum#507]
I've defined the following ruleset:

```yaml
extends: [[spectral:oas, recommended]]
rules:
  test-json-schema:
    type: validation
    description: "Test"
    message: "Test Message"
    howToFix: "TestHowToFix"
    given: $.paths[*][*]
    severity: error
    then:
      field: "tags"
      function: "schema"
      functionOptions:
        schema:
          type: "array"
          contains:
            pattern: "^(user|admin)$"
          minContains: 1
          maxContains: 1
``` 

All Vacuum reports, regardless of schema, are returning a validation error: 

> `schema` function needs a `schema` property to be supplied at a minimum: minimum property number not met (1)

I've tried this with multiple variations to the `schema` property of `functionOptions`, and all of them return the same error message, so this seems to be related to the `MinProperties` check that was added in 0.10.0.

**Comment by daveshanley** [comment]
Hmm... will look into this for you, not sure what is going on but something looks off.

**Comment by daveshanley** [comment]
Should be fixed in `v0.11.0`. This change was reverted as it was a mistake.
